src/tools/distillery.ml

```diff
@@ -246,8 +273,14 @@ let main () =
           template, name, dir
       | _ -> Arg.usage spec usage_msg; exit 1
     in
-    let env, preds, source_dir = init_project template name in
-    create_project ~name ~env ~preds ~source_dir ~destination_dir:destination_dir
+    if not (check_forbidden_project_name name template) then
```

Comment:
I would negate the output of `check_forbidden_project_name`. You should return `true` if what you check for (if the project name is forbidden) is true, no?